app/scripts/lib/app-start.js

```diff
@@ -588,6 +564,51 @@ define(function (require, exports, module) {
       }
     },
 
+    /**
+     * Check whether there are any problems accessing localStorage.
+     * Errors are logged to Sentry and internal metrics.
+     *
+     * If there is a problem accessing localStorage, the user
+     * will be redirected to `/cookies_disabled` from _selectStartPage
+     */
+    testLocalStorage: function () {
+      var self = this;
+      return p().then(function () {
+        // only test localStorage if the user is not already at
+        // the cookies_disabled screen.
+        if (! self._isAtCookiesDisabled()) {
+          self._storage.testLocalStorage(self._window);
+        }
+      }).fail(self.captureError.bind(self));
```

Comment:
Is there no way to move this and rely on the original `.fail` that we have above: https://github.com/mozilla/fxa-content-server/pull/3424/files#diff-b80f22cfd726dfdc2bb7ed3e82dc7264R105